.github/workflows/ci.yml

```diff
@@ -6,7 +6,7 @@ name: MATLAB Build
 on:
   # Triggers the workflow on push or pull request events but only for the main branch
   push:
-    branches: [ main ]
+    #branches: [ main ]
```

Comment:
intentionally left comment?

Reply:
No, it should be uncommented. I commented it before in order to qualify the job from the 'run-build-example' branch. I've updated in the latest commit.